src/gas.rs

```diff
@@ -28,33 +28,50 @@ impl GasMeterResult {
 #[derive(Debug)]
 /// Struct to keep track of gas usage
 pub struct GasMeter {
-    limit: Gas,
+    /// Initial amount of gas added by the transactor.
+    initial: Gas,
     /// Amount of gas left from initial gas limit. Can reach zero.
     gas_left: Gas,
 }
 
 impl GasMeter {
-    /// Creates a new `GasMeter` with given gas limits
-    pub fn with_limit(gas_limit: Gas) -> GasMeter {
+    /// Minimum amount of gas that has to be used in order to call a contract
+    /// execution.
+    // TODO: Add the correct value here that changes in respect to the transfer
+    // contract.
+    pub const MIN_TERMINATION_GAS_REQUIRED: Gas = 70440;
+
+    /// Creates a new `GasMeter` with given initial gas.
+    pub fn new(initial: Gas) -> GasMeter {
```

Comment:
Consider giving the same name as the source of this value:
https://github.com/dusk-network/phoenix-core/blob/master/src/fee.rs#L31